### ISSUE TITLE
[Accessibility][Status Message - Bot Responses-More option]: Screen reader doesn’t announce any confirmation on activating 'delete' menu item.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2050]
**User Experience:**
Screen reader users will get confused that whether the action is performed or not if user will not get any confirmation after executing that action.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. Select 'Bot Responses' link from left navigation.
3. Navigate within the table using tab key.
4. Press right arrow key & navigate up to 'More option' button & activate it.
5. Select 'Delete' menu item and observe the issue.

**Actual Result:**
Screen reader doesn’t announce any confirmation on activating 'delete' menu item.

**Note:**
Screen reader read twice 'button button' information.

**Expected Result:**
Screen reader should announce the confirmation on activating 'delete' menu item. 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/CELA/Official_Microsoft_Accessibility_Standards/MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx?d=wc154fbe5702f42fbb2bc87fbb3a23d91&csf=1&e=vba1jx

**Attachment:**
[MAS4.1.3_Bot responses_Delete.zip](https://github.com/microsoft/BotFramework-Composer/files/4239080/MAS4.1.3_Bot.responses_Delete.zip)

**Comment by beyackle** [comment]
Just a progress report for a bit more visibility on my process here. I've been trying to add an aria-live region in various places and none of them have worked; my current theory is that this is because they haven't been present on page load, so the browser doesn't start watching for them. I have another solution in the works that I'll see if I can make happen. It might take a little more time, but it'll be worth it in the end for some more flexibility it'll give us (being able announce anything from anywhere in the app).

**Comment by ashish315** [comment]
As checked the issue is fixed so HCL is closing the issue.
